File: Remainder on dividing by 11.cpp
class Solution
{

public:
    int xmod11(string x)
    {
        int len = x.length(); 
  
    int num, rem = 0; 
 
    for (int i = 0; i < len; i++) { 
        num = rem * 10 + (x[i] - '0'); 
        rem = num % 11; 
    } 
  
    return rem; 
    
    }
};

File: Remainder on dividing by 11.py
class Solution:
    def xmod11(self,x):
        l = len(x)
        num = 0
        rem = 0
        for i in range(0,len(x)):
            num = rem*10 + int(x[i])
            rem = num % 11
            
        return rem
            
        
